### ISSUE TITLE
Verify electrum download

### ISSUE THREAD

**Issue by emilio888** [pooler/electrum-ltc#103]
Hi, I'm a newbie, how do I verify the electrum file that I downloaded? I tried downloading Tor but it's not opening and the steps on the website arent't clear. Any help would be appreciated.

**Comment by pooler** [comment]
Which part of the steps did you find not clear?
The procedure is the same as shown [here](https://www.torproject.org/docs/verifying-signatures.html.en), you just need to replace the key and the filenames. You do not need Tor to verify Electrum-LTC.

```
gpg --keyserver pool.sks-keyservers.net --recv-keys 0x6fc4c9f7f1be8fea
gpg --fingerprint 0x6fc4c9f7f1be8fea
gpg --verify Electrum-LTC-2.8.3.5.tar.gz.asc Electrum-LTC-2.8.3.5.tar.gz
```

**Comment by emilio888** [comment]
Where is the asc file ? It didn't come with the download.

**Comment by pooler** [comment]
> Where is the asc file ? It didn't come with the download.

You must download it separately from the Electrum-LTC website.

**Comment by 7022Andre** [comment]
@pooler Is the key that you posted in this issue also posted somewhere else on [the electrum LTC website](http://electrum-ltc.org) or in this GitHub repo?
I agree with @emilio888 that the steps are not clear for somebody who's new by just visiting the [Tor Project site](https://www.torproject.org/docs/verifying-signatures.html.en).

I can write a quick blurb on how-to verify the digital signature and create a pull request for this repo. Let me know if you want me to.

**Comment by pooler** [comment]
> @pooler Is the key that you posted in this issue also posted somewhere else on the electrum LTC website or in this GitHub repo?

Yes, there are links to it next to every download on the website.

> I can write a quick blurb on how-to verify the digital signature and create a pull request for this repo. Let me know if you want me to.

Thanks, but maybe post it here instead, or alternatively send me an email. I can then have it uploaded to the website.

**Comment by 7022Andre** [comment]
Here you go: [markdown styled file for download](https://github.com/pooler/electrum-ltc/files/1605160/verify_signature.txt)
-----
# Always verify the digital signatures of the files you download!

**Download accompanied signature file (.asc)**
- Linux:
  - tar.gz: https://electrum-ltc.org/download/Electrum-LTC-2.9.3.1.tar.gz.asc
  - .zip: https://electrum-ltc.org/download/Electrum-LTC-2.9.3.1.zip.asc
- Mac OS X:
  - https://electrum-ltc.org/download/electrum-ltc-2.9.3.1.dmg.asc
- Windows:
  - standalone: https://electrum-ltc.org/download/electrum-ltc-2.9.3.1.exe.asc
  - installer: https://electrum-ltc.org/download/electrum-ltc-2.9.3.1-setup.exe.asc
  - portable: https://electrum-ltc.org/download/electrum-ltc-2.9.3.1-portable.exe.asc

Use the below instructions if you're using Linux. Visit [Tor Project](https://www.torproject.org/docs/verifying-signatures.html.en) for instructions for Windows and OS X. The signature key is `0x6fc4c9f7f1be8fea`.

**Import signing key from keyserver**

`gpg --keyserver pool.sks-keyservers.net --recv-keys 0x6fc4c9f7f1be8fea`

You should see:
```
gpg: key 6FC4C9F7F1BE8FEA: public key "pooler <pooler@litecoinpool.org>" imported
gpg: marginals needed: 3  completes needed: 1  trust model: pgp
gpg: depth: 0  valid:   2  signed:   2  trust: 0-, 0q, 0n, 0m, 0f, 2u
gpg: depth: 1  valid:   2  signed:   0  trust: 0-, 0q, 0n, 0m, 2f, 0u
gpg: Total number processed: 1
gpg:               imported: 1
```

**Verify that the imported fingerprint is correct**

`gpg --fingerprint 0x6fc4c9f7f1be8fea`

You should see:
```
pub   rsa2048 2013-07-21 [SC]
      CAE1 092A D355 3FFD 21C0  5DE3 6FC4 C9F7 F1BE 8FEA
uid           [ unknown] pooler <pooler@litecoinpool.org>
sub   rsa2048 2013-07-21 [E]
```

**Verify signature of downloaded file**
`gpg --verify Electrum-LTC-2.9.3.1.tar.gz.asc Electrum-LTC-2.9.3.1.tar.gz`

The output should say "Good signature":
```
gpg: Signature made Thu 17 Aug 2017 10:59:17 AM PDT
gpg:                using RSA key 6FC4C9F7F1BE8FEA
gpg: Good signature from "pooler <pooler@litecoinpool.org>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: CAE1 092A D355 3FFD 21C0  5DE3 6FC4 C9F7 F1BE 8FEA
```

Notice that there is a warning because you haven't assigned a trust index to this person.
```
gpg: WARNING: This key is not certified with a trusted signature!
gpg: There is no indication that the signature belongs to the owner.
```
This means that GnuPG verified that the key made that signature, but it's up to you to decide if that key really belongs to the developer. The best method is to meet the developer in person and exchange key fingerprints.

**Comment by pooler** [comment]
@7022Andre: Thanks! I've added a little more information and uploaded the instructions [here](https://electrum-ltc.org/verifying-signatures.html), and I've linked to this page from the download page.

**Comment by allanster** [comment]
Have run this multiple times and does not match what instructions on site say I should see...

sh-3.2# gpg --keyserver pool.sks-keyservers.net --recv-keys 0x6fc4c9f7f1be8fea
gpg: key 6FC4C9F7F1BE8FEA: 9 signatures not checked due to missing keys
gpg: key 6FC4C9F7F1BE8FEA: "pooler <pooler@litecoinpool.org>" not changed
gpg: Total number processed: 1
gpg:              unchanged: 1

**Comment by allanster** [comment]
Concerned that this is reporting ALL are missing, should I not be?...

gpg --keyserver pool.sks-keyservers.net --recv-keys 0x6fc4c9f7f1be8fea

sh-3.2# gpg --keyserver pool.sks-keyservers.net --recv-keys 0x6fc4c9f7f1be8fea
gpg: key 6FC4C9F7F1BE8FEA: 9 signatures not checked due to missing keys
gpg: key 6FC4C9F7F1BE8FEA: "pooler <pooler@litecoinpool.org>" not changed
gpg: Total number processed: 1
gpg:              unchanged: 1
